parser/formattext.go

```diff
@@ -98,17 +97,19 @@ func (fc *FontConfig) FormatText(text string, maxWidth int, cursorOverlapWidth i
 			// it could span multiple words. The true solution would require optimistically trying to fit all
 			// remaining words onto the same line, rather than only looking at the current word + cursor. However,
 			// this is "good enough" and likely works for almost all actual use cases in practice.
-			if len(nextWord) > 0 && (!isFirstLine || fc.isParagraphBreak(nextWord)) {
+			if len(nextWord) > 0 && (fc.isFirstLine(curLineNum) == false || fc.isParagraphBreak(nextWord)) {
```

Comment:
nit: instead of explicitly checking for `false`, just do `!fc.isFirstLine(curLineNum)`

Reply:
https://github.com/huderlem/poryscript/pull/59/commits/5e52164eeb5158fc8dade9a4bc5b2300842ac47c